### DISCUSSION TITLE
676dd3b3d3bf83a38de45336

### DISCUSSION THREAD

**Discussion by giscus** [Admin-ToxicGames/toxicgames#6] [General]
# 676dd3b3d3bf83a38de45336

Discover a wide range of cracked games, mods, and gaming resources for PC, Mac, and more. Download your favorite games for free and enjoy endless gaming adventures!

https://toxicgames.in/676dd3b3d3bf83a38de45336

**Comment by Admin-ToxicGames** [comment]
So, to change the language there are 2 ways:

### **First way**  
First, download the Language Changer from this [direct link](https://vk.com/s/v1/doc/-IcwfrS3SRXTqmzlEp2_q2laEyWVWlwJQh7EyZwJDbW4C0XcDh4) or from [Mediafire](https://www.mediafire.com/file/lt5eveuyz580p8x/Language+Changer+(Mirror%27s+Edge+Catalyst).pkg/file).

### **Second way**  
1. Open this file in a text editor:  
`/Applications > MirrorsEdgeCatalyst.app > Contents > SharedSupport > prefix > drive_c > Program Files(x86) > Mirrors Edge Catalyst/ CPY.ini`

2. In the `CPY.ini` file, change `Language=ru_RU` to `Language=en_US`.

It will work.
